civictechprojects/models.py

```diff
@@ -886,8 +886,12 @@ def get_owner(self):
     @staticmethod
     def from_json(owner, file_category, file_json):
         file_name_parts = file_json['fileName'].split('.')
-        file_name = "".join(file_name_parts[:-1])
```

Comment:
As I talked about during the meeting, the original code is excluding the last element and including everything else into the file_name field. It will cause a problem when a file has only a filename, making file_name to be an empty string.